dex-it/src/test/scala/com/wavesplatform/it/api/DexApi.scala

```diff
@@ -161,6 +165,7 @@ object DexApi {
       override def tryPlace(order: Order): F[Either[MatcherError, MatcherResponse]] = tryParseJson {
         sttp
           .post(uri"$apiUri/orderbook")
+          .readTimeout(3.minutes) // TODO find way to decrease timeout!
```

Comment:
Is this still required (and other `readTimeout` too) ?

Reply:
Yes